docs/conda.rst

```diff
@@ -0,0 +1,275 @@
+.. inclusion-marker-start-do-not-remove
+
+Building a Conda environment with GPU support for Horovod
```

Comment:
I recommend title case & imperative here for consistency with the other docs in the TOC: "Build a Conda Environment with GPU Support for Horovod"

Reply:
Agreed.